File: HelloGoodbye.py
import sys

if __name__ == "__main__":
    if len(sys.argv) > 2:
        print(f"Hello {sys.argv[1]} and {sys.argv[2]}")
        print(f"Goodbye {sys.argv[2]} and {sys.argv[1]}")
    else:
        print("Usage: HelloGoodbye name-1 and name-2")

File: HelloGoodbye.java
/**
 * Example  of using  Java command line arguments based. This example is the 
 * first assignment from Algorithms Part I, lectured by Princeton University 
 * me on Coursera. 
 * 
 * One can enroll in this course through coursera.org/learn/algorithms-part1.
 * 
 */
public class HelloGoodbye {
    public static void main(String[] args) {
        if (args.length > 1) {
        System.out.println("Hello " + args[0] + " and " + args[1]);
        System.out.println("Goodbye " + args[1] + " and " + args[0]);
        } else {
            System.out.println("Usage: java HelloGoodbye Name1 Name2");
        }
    }
}
